src/ConfigGen/Program.cs

```diff
@@ -418,26 +418,44 @@ static string createClassHeader(string name, string docs, bool derive)
             return codeText;
         }
 
-        static string createConsumerSpecific()
+        static string createClassConstructors(string name)
         {
-            return
-@"        /// <summary>
-        ///     Initialize a new empty <see cref=""ConsumerConfig"" /> instance.
+            var codeText = $@"
+        /// <summary>

```

Comment:
It was probably me who caused you to put this in here, but I don't think `CopyFrom` is necessary.